core/src/main/java/com/linecorp/armeria/server/ServerConfig.java

```diff
@@ -214,8 +214,16 @@ public interface ServerConfig {
 
     /**
      * Returns whether the worker {@link Executor} is shut down when the {@link Server} stops.
+     *
+     * @deprecated This method is not used anymore. The {@code blockingTaskExecutor} is shut down if
+     *             the {@code shutdownOnStop} of
+     *             {@link ServerBuilder#blockingTaskExecutor(ScheduledExecutorService, boolean)}
+     *             is set with {@code true}.
```

Comment:
I'm not sure if it's a good reason to deprecate this method. A user will lose the ability to tell if he or she configured the server to shut it down on stop. It would also make people wonder why this method is deprecated while `shutdownWorkerGroupOnStop` is not.

Reply:
I thought users do not care about the `shutdownOnStop` flag after they specify it to the builder.
I tried to convince myself that a user might need this API but I could find a reason for that. 
I believed that we should've made those APIs package-private from the first so that they are hidden from users.

> It would also make people wonder why this method is deprecated while shutdownWorkerGroupOnStop is not.

That's a good point. If you worry about this, how about deprecating the API as well? Later, we can just remove the `public` modifier not to expose it anymore. 🤔

Reply:
Yeah, we can do that, too.